### DISCUSSION TITLE
Happy new user

### DISCUSSION THREAD

**Discussion by rountree** [zellij-org/zellij#3900] [Show and tell]
TL;DR:  THANK YOU.

I just heard zellij yesterday, and today I've managed to modify the configuration file that let's me keep my tmux muscle memory and solve a seemingly intractable problem with how tmux handles control keys.

For decades, I've run into the problem of holding down the control key slightly too long when flying around in tmux.  After staring at zellij's configuration file for no more than 20 minutes, I've got the following working:

```
    tmux {
        bind "," { SwitchToMode "renametab"; }
        bind "[" { SwitchToMode "scroll"; }
        bind "Ctrl a" { Write 1; SwitchToMode "normal"; }
        bind "c" { NewTab; SwitchToMode "normal"; }
        bind "n" { GoToNextTab; SwitchToMode "normal"; }
        bind "Ctrl n" { GoToNextTab; SwitchToMode "normal"; }
        bind "p" { GoToPreviousTab; SwitchToMode "normal"; }
        bind "Ctrl p" { GoToPreviousTab; SwitchToMode "normal"; }
    }
```
```
    shared_except "locked" "tab" {
        bind "Ctrl t" { SwitchToMode "tab"; }
        bind "Ctrl 1" { GoToTab 1; SwitchToMode "normal"; }
        bind "Ctrl 2" { GoToTab 2; SwitchToMode "normal"; }
        bind "Ctrl 3" { GoToTab 3; SwitchToMode "normal"; }
        bind "Ctrl 4" { GoToTab 4; SwitchToMode "normal"; }
        bind "Ctrl 5" { GoToTab 5; SwitchToMode "normal"; }
        bind "Ctrl 6" { GoToTab 6; SwitchToMode "normal"; }
        bind "Ctrl 7" { GoToTab 7; SwitchToMode "normal"; }
        bind "Ctrl 8" { GoToTab 8; SwitchToMode "normal"; }
        bind "Ctrl 9" { GoToTab 9; SwitchToMode "normal"; }
        bind "Ctrl 0" { GoToTab 10; SwitchToMode "normal"; }
    }
```
Weirdly enough, the `Ctrl 0` binding doesn't work, but since the number starts at 1, I'm not minding too much (and I have some Googling to do to see if this is a known limitation).

I've been using screen/byobu/tmux for thirty years at this point, and I've been teaching tmux to my summer interns for the past fifteen.  I'm not going to say that zellij is going to replace tmux at work and on my macs, but I can see getting to that point by the end of next week.

Thanks for the excellent work, particularly in making the configuration file so discoverable.

**Comment by imsnif** [comment]
I think the `Ctrl 0` issue is because your terminal emulator is intercepting that key as "reset font size to default". As in `Ctrl +` and `Ctrl -` to increase/decrease font size and `Ctrl 0` to reset it.

At least that's how it was for me with Alacritty.

**Reply by rountree** [reply]
Yep, that's the issue.  Thanks!

**Comment by stdedos** [comment]
Thank you rountree for posting your "compatibility" layer here!

Here are some byobu compatibility as well:

```kdl
    shared_among "normal" "locked" {
        bind "F2" { NewTab; }
        bind "F3" { GoToPreviousTab; }
        bind "F4" { GoToNextTab; }
        // No need:
        // https://zellij.dev/documentation/configuration#how-do-i-update-the-config-file-for-running-sessions
        // bind "F5" { ReloadConfiguration; }
        bind "F6" { Detach; }
        bind "F7" { SwitchToMode "Scroll"; }
        bind "F8" { SwitchToMode "RenameTab"; TabNameInput 0; }
        bind "F9" {
            LaunchOrFocusPlugin "configuration" {
                floating true
                move_to_focused_tab true
            };
        }

        bind "Shift F2" { NewPane "Down"; }
        bind "Ctrl F2" { NewPane "Right"; }
        bind "Shift F3" { MoveFocus "Left"; }
        bind "Shift F4" { MoveFocus "Right"; }
        bind "Shift F5" { CloseFocus; }
        bind "Ctrl F6" { Detach; }
        bind "Alt PageUp" { SwitchToMode "Scroll"; }
        bind "Alt PageDown" { SwitchToMode "Scroll"; }
    }
```

(personal preference `shared_among "normal" "locked"` or `normal`)
